starter_code/queries.md

```diff
@@ -4,43 +4,84 @@
 
 ### 1. All the companies whose name match 'Babelgum'. Retrieve only their `name` field.
 
-<!-- Your Code Goes Here -->
+query: {name: "Babelgum"}
+projection: 
+sort: 
+skip: 
+limit: 
 
 ### 2. All the companies that have more than 5000 employees. Limit the search to 20 companies and sort them by **number of employees**.
 
-<!-- Your Code Goes Here -->
+query: {number_of_employees: {$gt: 5000}}
+projection: 
+sort: {number_of_employees: 1}
+skip: 
+limit: 20
 
 ### 3. All the companies founded between 2000 and 2005, both years included. Retrieve only the `name` and `founded_year` fields.
 
-<!-- Your Code Goes Here -->
+query: {$and: [{founded_year: {$gte: 2000}}, {founded_year: {$lte: 2005}}]}
```

Comment:
That is one way of doing it, another shorter way for the same would be: 

{founded_year: {$gt : 1999, $lt: 2006}}